gulpfile.js

```diff
@@ -131,28 +137,31 @@ gulp.task('package:online', ['clean'], () => {
 });
 
 gulp.task('package:offline', ['clean'], () => {
-    var json = JSON.parse(fs.readFileSync('package.json'));
-    var name = json.name;
-    var version = json.version;
+    util.setExtensionPath(__dirname);
+
+    var packageJSON = getPackageJSON();
+    var name = packageJSON.name;
+    var version = packageJSON.version;
     var packageName = name + '.' + version;
 
     var packages = [];
-    packages.push({ rid: 'win7-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.Windows }, { flavor: Flavor.Desktop, coreClrFlavor: CoreClrFlavor.Windows }] });
-    packages.push({ rid: 'osx.10.11-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.OSX }] });
-    packages.push({ rid: 'centos.7-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.CentOS }] });
-    packages.push({ rid: 'debian.8-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.Debian }] });
-    packages.push({ rid: 'fedora.23-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.Fedora }] });
-    packages.push({ rid: 'opensuse.13.2-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.OpenSUSE }] });
-    packages.push({ rid: 'rhel.7.2-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.RHEL }] });
-    packages.push({ rid: 'ubuntu.14.04-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.Ubuntu14 }] });
-    packages.push({ rid: 'ubuntu.16.04-x64', omnisharps: [{ flavor: Flavor.CoreCLR, coreClrFlavor: CoreClrFlavor.Ubuntu16 }] });
+    packages.push({ rid: 'win7-x64', platform: new PlatformInformation('win32') });
```

Comment:
in that case, I'm not sure there's a lot of value. I don't think anyone is really using this offline installers anyway.